schema.sql

```diff
@@ -0,0 +1,287 @@
+    FOREIGN KEY (pot_factory_id) REFERENCES pot_factory (id),
+    FOREIGN KEY (whitelisted_deployer_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot (
+    id VARCHAR PRIMARY KEY,
+    pot_factory_id INT NOT NULL,
+    deployer_id VARCHAR NOT NULL,
+    deployed_at TIMESTAMP NOT NULL,
+    source_metadata VARCHAR NOT NULL,
+    owner_id VARCHAR NOT NULL,
+    chef_id VARCHAR NULL,
+    name TEXT NOT NULL,
+    description TEXT NOT NULL,
+    max_approved_applicants INT NOT NULL,
+    base_currency VARCHAR NULL,
+    application_start TIMESTAMP NOT NULL,
+    application_end TIMESTAMP NOT NULL,
+    matching_round_start TIMESTAMP NOT NULL,
+    matching_round_end TIMESTAMP NOT NULL,
+    registry_provider VARCHAR NULL,
+    min_matching_pool_donation_amount VARCHAR NOT NULL,
+    sybil_wrapper_provider VARCHAR NULL,
+    custom_sybil_checks VARCHAR NULL,
+    custom_min_threshold_score INT NULL,
+    referral_fee_matching_pool_basis_points INT NOT NULL,
+    referral_fee_public_round_basis_points INT NOT NULL,
+    chef_fee_basis_points INT NOT NULL,
+    total_matching_pool VARCHAR NOT NULL,
+    total_matching_pool_usd DECIMAL(10, 2) NULL,
+    matching_pool_balance VARCHAR NOT NULL,
+    matching_pool_donations_count INT NOT NULL,
+    total_public_donations VARCHAR NOT NULL,
+    total_public_donations_usd DECIMAL(10, 2) NULL,
+    public_donations_count INT NOT NULL,
+    cooldown_end TIMESTAMP NULL,
+    all_paid_out BOOLEAN NOT NULL,
+    protocol_config_provider VARCHAR NULL,
+    FOREIGN KEY (pot_factory_id) REFERENCES pot_factory (id),
+    FOREIGN KEY (deployer_id) REFERENCES account (id),
+    FOREIGN KEY (owner_id) REFERENCES account (id),
+    FOREIGN KEY (chef_id) REFERENCES account (id),
+    FOREIGN KEY (base_currency) REFERENCES account (id)
+  );
+
+-- Table pot_application
+CREATE TABLE
+  pot_application (
+    id SERIAL PRIMARY KEY,
+    pot_id INT NOT NULL,
+    applicant_id VARCHAR NOT NULL,
+    message TEXT,
+    current_status ENUM('Pending', 'Approved', 'Rejected', 'InReview') NOT NULL,
+    submitted_at TIMESTAMP NOT NULL,
+    last_updated_at TIMESTAMP,
+    tx_hash VARCHAR,
+    FOREIGN KEY (pot_id) REFERENCES pot (id),
+    FOREIGN KEY (applicant_id) REFERENCES account (id)
+  );
+
+-- Table pot_application_review
+CREATE TABLE
+  pot_application_review (
+    id SERIAL PRIMARY KEY,
+    application_id INT NOT NULL,
+    reviewer_id VARCHAR NOT NULL,
+    notes TEXT,
+    status ENUM('Pending', 'Approved', 'Rejected', 'InReview') NOT NULL,
+    reviewed_at TIMESTAMP NOT NULL,
+    tx_hash VARCHAR,
+    FOREIGN KEY (application_id) REFERENCES pot_application (id),
+    FOREIGN KEY (reviewer_id) REFERENCES account (id)
+  );
+
+-- Table pot_payout
+CREATE TABLE
+  pot_payout (
+    id SERIAL PRIMARY KEY,
+    recipient_id VARCHAR NOT NULL,
+    amount VARCHAR NOT NULL,
+    amount_usd DECIMAL(10, 2),
+    ft_id VARCHAR NOT NULL,
+    paid_at TIMESTAMP,
+    tx_hash VARCHAR,
```

Comment:
add `NOT NULL`

not actually sure that we need this, it's a little confusing as it could be the hash when the payout is set, or the hash when it is paid out